src/JsonSchema/Constraints/UndefinedConstraint.php

```diff
@@ -66,7 +66,8 @@ public function validateTypes(&$value, $schema = null, JsonPointer $path, $i = n
         }
 
         // check object
-        if ($this->getTypeCheck()->isObject($value)) {
+        if (TypeCheck\LooseTypeCheck::isObject($value)) { // Fixes failing assoc tests for default values - currently investigating
+        //if ($this->getTypeCheck()->isObject($value)) {  // to find the root cause of this, noting all other assoc tests pass.
```

Comment:
Thanks :-). I don't suppose anyone involved with the addition of that code is still around? Looks like it was done midway through last year.